app/src/main/java/mozilla/lockbox/view/ItemMutationFragment.kt

```diff
@@ -55,6 +53,15 @@ open class ItemMutationFragment : BackableFragment(), ItemMutationView {
     override val passwordChanged: Observable<String>
         get() = view!!.inputPassword.textChanges().map { it.toString() }
 
+    override val hostnameFocus
+        get() = focusChanges(view!!.inputHostname)
```

Comment:
Can we use `view?.inputHostname` here?

Reply:
(Same with pwd and username)

Reply:
No. This is a well established pattern.

Additionally, passing an optional to `focusChanges` means we get an optional back. We should fail fast here if there is a problem with the `layout.xml`.